src/dmd/frontend.h

```diff
@@ -2655,7 +2655,6 @@ enum class PURE : uint8_t
     fwdref = 1u,
     weak = 2u,
     const_ = 3u,
-    strong = 4u,
```

Comment:
Please also update the relevant header

Reply:
That's dmd/mtype.h, right? Removed it from there.